drive/modules/listener/Module.md

```diff
@@ -0,0 +1,122 @@
+## Overview
+
+This module generates a notification when the following events occur in the drive.
+- Creation of a new file
+- The creation of a new file in a specific folder
+- Creation of a new folder
+- Creation of a new folder in a specific folder
+- File update
+- File deletion
+- File deletion in a specific folder
+- Folder deletion
+- Folder deletion in a specific folder
+
+This module supports [Google Drive API v3](https://developers.google.com/drive/api).
+
+Before using this connector in your Ballerina application, complete the following:
+
+### Prerequisites
+- [Create a Google account](https://accounts.google.com/signup/v2/webcreateaccount?utm_source=ga-ob-search&utm_medium=google-account&flowName=GlifWebSignIn&flowEntry=SignUp)
+* [Domain used in the callback URL needs to be registered in google console as a verified domain.](https://console.cloud.google.com/apis/credentials/domainverification)
+If you are running locally, provide your ngrok url for domain verification. Then you are able to download a HTML file (e.g : google2c627a893434d90e.html). Copy the content of that HTML file and provide it as a configuration (i.e., via the domainVerificationFileContent parameter) to listener initialization.
+
+If you fail to verify or set up, see documentation for domain verification process.
+
+- Obtain tokens - Follow [this link](https://developers.google.com/identity/protocols/oauth2)
+
+## Quickstart
+
+To use the Google Calendar connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import listener
+Import the Google Drive Listener module to your Ballerina program as follows.
+i.e: The client connecter is imported to create a file in the drive here.
```

Comment:
Shall we remove the above line please? We are just listening to the create event.